agent/main.go

```diff
@@ -188,7 +188,7 @@ func mainImpl() {
 		log.Fatal(err)
 	}
 
-	if *wcToken == "" {
+	if *featureInstall && *wcToken == "" {
```

Comment:
hmm, should not this be `!featureInstall`? But I understood if its a feature then token is not needed and we should not log.fatal then?

Reply:
If we want to install the agents and the token hasn't been specified, we log.Fatal() the error (and exit).